### PR TITLE
Set up tests

### PR DESCRIPTION
**Pull request by nderkim**
Get tests working again

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,7 +12,8 @@
     "lint:fix": "eslint --fix .",
     "lint": "eslint .",
     "postinstall": "preconstruct dev && manypkg check",
-    "typecheck": "yarn postinstall && ultra -r tsc --noEmit"
+    "typecheck": "yarn postinstall && ultra -r tsc --noEmit",
+    "test": "jest"
```
Comment (Thinkscape) [inline package.json:16]: How about:
```
"test:unit": "jest",
"test": "yarn run test:unit && yarn run typecheck",
```

I like my `test` command to test all-the-things without me having to consider what it tests and what it doesn't.

Comment (nderkim) [reply package.json]: done ✅

File: package.json [hunk 2]
```diff
@@ -12,7 +12,9 @@
     "lint:fix": "yarn run lint --fix",
     "lint": "eslint . --ignore-path \"./.gitignore\"",
     "postinstall": "preconstruct dev && manypkg check",
-    "typecheck": "yarn postinstall && ultra -r tsc --noEmit"
+    "typecheck": "tsc --noEmit",
```
Comment (lukebennett88) [inline package.json:15]: I think we should keep the `postinstall` here. If you were to run `yarn build` before running `yarn typecheck` you will get a bunch of false positive type errors.

Comment (nderkim) [reply package.json:15]: I think the type errors are valid errors. eg. `level` is declared as a `number` but is being passing in as a `string` throughout the codebase.
Screen Shot 2022-04-08 at 8 48 00 am

Comment (lukebennett88) [reply package.json:15]: If you run `preconstruct dev` it should be a number ¯\_(ツ)_/¯ :
PixelSnap 2022-04-08 at 08 57 07@2x

**Review by freemany — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (nderkim) [reply package.json:15]: @Thinkscape looks like there is only one `tsconfig` in this repo, so only one process to run

**Review by lukebennett88 — APPROVED** [review]

**Pull request merged**